scripts/check_plugins_location.sh

```diff
@@ -15,7 +15,7 @@
 set -e
 
 # shellcheck source=./scripts/util.sh
-source ./util.sh
+source $(dirname "$0")/util.sh
```

Comment:
Looks like a job for good old `$(cd "$(dirname "$0")"; pwd)` to avoid relative path stuff.

Reply:
Nope, shellcheck complains about missing quotes around the path.